ufs_srweather_app_meta.h.in

```diff
@@ -0,0 +1,24 @@
+/*
+ * Meta information for ufs-srweather-app which can be used by other
+ * packages.
+ *
+ * This file is automatically generated by the build system
```

Comment:
If this file is automatically generated, should it be committed to the repository?

Reply:
No, that comments is for the autogenerated file, which is generated with  ufs_srweather_app_meta.h.in